projects/quantlib/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://www.quantlib.org/"
+language: c++
+# TODO: Get primary author dets
+primary_contact: ""
```

Comment:
@lballabio is it ok if I put the gmail account listed on your github profile here, or would you prefer a different contact?

Reply:
The address of the QuantLib users mailing list would be best.

Reply:
Got a couple of quetions about that;
- Are you sure? this email will receive bug-reports about potential security vulnerabilities. So if it get's sent to a public mailing list all vulnerabilities will be immediately public.
- Is the mailing list email address a gmail/google account, [it is one of the requirements for getting access to the oss-fuzz dashboard](https://google.github.io/oss-fuzz/faq/#why-do-you-require-a-google-account-for-authentication).

Reply:
Ok, use my gmail address then.

Reply:
done